project/make/.dockerinit-gccgo

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+IAMSTATIC="true"
+source "$(dirname "$BASH_SOURCE")/.go-autogen"
+
+go build --compiler=gccgo \
+	-o "$DEST/dockerinit-$VERSION" \
+	"${BUILDFLAGS[@]}" \
+	--gccgoflags "
+		-g
+		-static -Wl,--no-export-dynamic
```

Comment:
'-static' is already defined by EXTLDFLAGS_STATIC_DOCKER, so we can remove it from here just like th gccgo script